Makefile

```diff
@@ -373,3 +373,10 @@ release-push-%:
 	$(MAKE) clean-bin
 	$(MAKE) ARCH=$* build
 	$(MAKE) ARCH=$* push
+
+# SvCat Kubectl plugin stuff
+############################
+svcat: $(BINDIR)/svcat
+$(BINDIR)/svcat: .init .generate_files cmd/svcat/main.go
```

Comment:
More needs to be done here so that when I modify a file, like ./cmd/svcat/class/getCmd.go, then run `make svcat` that it doesn't immediately stop because the binary exists.

```console
$ make svcat
make: Nothing to be done for `svcat'.
```

Reply:
just make it so that everything in cmd/svccat is a dependency for the exe

Reply:
I still don't understand why we're forcing make to do this when the go toolset understands caching and dependencies.

Reply:
@MHBauer Would switching this to a phony target be acceptable?

Reply:
To me personally, all of the go builds, yes. Last complainant was msft saying that was too slow, hence the crazy `find *.go` timestamp stuff.

Reply:
Since the docker image mounts go/pkg, go's cache should persist from build to build. My vote is to use a phony target in this PR and maybe doing a follow-on to switch to that for the others. 😀

Reply:
I'm fine with this. Do this one, follow up for others.

Reply:
Added the phony target.